addon/services/rollbar.js

```diff
@@ -5,20 +5,20 @@ import deepMerge from 'lodash.merge';
 import Rollbar from 'rollbar';
 
 export default Service.extend({
-  enabled: computed({
+  enabled: computed('config.enabled', {
     get() {
-      return this.get('config.enabled');
+      return this.config.enabled
```

Comment:
See [no-get](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-get.md)

Reply:
That was for compatibility issues :) Can we revert all changes to files not related to update?